onadata/libs/utils/cache_tools.py

```diff
@@ -47,3 +50,20 @@ def safe_delete(key):
 def safe_key(key):
     """Return a hashed key."""
     return hashlib.sha256(force_bytes(key)).hexdigest()
+
+
+def reset_project_cache(project, request):
+    """
+    Clears and sets project cache
+    """
+    from onadata.libs.serializers.project_serializer import ProjectSerializer
```

Comment:
I'm going to assume we had to import here because of cyclic import error, no?

Reply:
Yes, this is here to avoid a cyclic import error